src/components/CategoriesSection/CategoriesSection.tsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from 'react-router-dom';
+import styles from './styles.module.scss';
+import { DeviceCategory, Paths } from '@/enums';
+import { Product } from '@/types';
+import { FC } from 'react';
+import { getQuantityByCategory } from '@/utils/getQuantityByCategory';
+
+interface CategoriesSectionProps {
+  products: Product[];
+}
+
+export const CategoriesSection: FC<CategoriesSectionProps> = ({ products }) => {
+  return (
+    <>
+      <h2 className={styles.title}>Shop by category</h2>
+
+      <section className={styles.container}>
```

Comment:
The `<section>` HTML element represents a generic standalone section of a document, which doesn't have a more specific semantic element to represent it. Sections should always have a heading, with very few exceptions. [(c)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

So, title should be inside